Atomic/atomic.py

```diff
@@ -1113,6 +1114,29 @@ def set_debug(self):
         if self.args.debug:
             self.debug = True
 
+    def get_last_time_all_scanned(self):
+        """
+        Will read through the scan_summary.json file and get the most recent scan's time.
+        """
+        try:
```

Comment:
That can be done but one of them is to get the vulnerable ids and the idea of this function is to specifically for the most recent scan time. I can look into modifying it if necessary.